src/Nethermind/Nethermind.Evm.Test/TransactionProcessorTests.cs

```diff
@@ -458,7 +458,7 @@ public void Can_estimate_with_destroy_refund_and_below_intrinsic_pre_berlin()
             tracer.CalculateAdditionalGasRequired(tx, releaseSpec).Should().Be(24080);
             tracer.GasSpent.Should().Be(35228L);
             long estimate = estimator.Estimate(tx, block.Header, tracer);
-            estimate.Should().Be(59307);
```

Comment:
I cannot fully figure out why this estimations are higher than what we had before.